website/source/guides/governance-and-policy/quotas.html.md

```diff
@@ -65,13 +65,18 @@ limit {
     region_limit {
         cpu = 2500
         memory = 1000
+		networks = [
+			{ mbits = 50 }
+		]
     }
 }
 ```
 
 A quota specification is composed of one or more resource limits. Each limit
 applies to a particular Nomad region. Within the limit object, operators can
-specify the allowed CPU and memory usage.
+specify the allowed CPU, memory usage, and network bandwidth. Network bandwidth
+limits may only specify a single limit for all interfaces. Network quotas are
```

Comment:
How about `Network quota limits were introduced in Nomad 0.10.2, and are optional. They will not be enforced if omitted`